### ISSUE TITLE
Probability Maps

### ISSUE THREAD

**Issue by MiguelAngeloMartinsRibeiro** [dbolya/yolact#713]
Hi @dbolya 
Is it possible to get the probability maps, i.e, instead of binary masks get a matrix with the probability of each pixel?

Thank you for your time

**Comment by MiguelAngeloMartinsRibeiro** [comment]
The softmax output would be useful as well

**Comment by telemetrieTP23** [comment]
You are right the probability map Output would really be nice for Heatmap Visualisation of a specific class.
Convert the Modell to ONNX and visualize it with https://netron.app/ 
So you can find the specific layers you need.
You can also use pre converted ONNX yolact/yolactEdge Modells from https://github.com/PINTO0309/PINTO_model_zoo

If you use Python then you can use this as starting point for Postprocessing Layers: https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/instance_segmentation_demo/python

For c++ use this:
https://github.com/PINTO0309/PINTO_model_zoo/blob/main/085_Yolact_Edge/30_Full_Converted_mbnv2_550x550/yolact_test.cpp
